SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -25,10 +46,8 @@ class SimplenoteUISmokeTestsSearch: XCTestCase {
         Sidebar.open()
         Sidebar.tagsDeleteAll()
         NoteList.openAllNotes()
-        NoteList.createNoteAndLeaveEditor(noteName: godzillaNoteName + "\n\n" + godzillaInfo, tags: ["sea-monster", "reptile", "prehistoric"])
-        NoteList.createNoteAndLeaveEditor(noteName: kingKongNoteName + "\n\n" + kingKongInfo, tags: ["ape", "prehistoric"])
-        NoteList.createNoteAndLeaveEditor(noteName: mechagodzillaNoteName + "\n\n" + mechagodzillaInfo, tags: ["man-made", "robot"])
-        NoteList.createNoteAndLeaveEditor(noteName: diacriticNoteName + "\n\n" + diacriticInfo, tags: ["language", "diacritic"])
+
+        allNotes.forEach { NoteList.createNoteThenLeaveEditor($0) }
```

Comment:
Double advantage here.

1. Using a custom API that takes `NoteData` as the input, we can DRY the code to create a new note at the test level.
2. Since we have a type for the notes, we can use an array and call `forEach` instead of writing a call of same method for each different note.